### DISCUSSION TITLE
How to Override struts.multipart.maxStringLength=4096 in Struts 1.5.0-RC2 Maven Project?

### DISCUSSION THREAD

**Discussion by MarcSchiering** [ste-gr/struts1-reloaded#44] [General]
We are currently working on a Maven-based project using Struts version 1.5.0-RC2, and we need to override the property struts.multipart.maxStringLength=4096. Our goal is to increase this value to allow for larger multipart data handling. However, we are not sure of the best way to override this setting in Struts 1.

Could someone provide guidance on how to properly configure or override this property in a Struts 1 Maven project?

Any help would be greatly appreciated!

Thank you in advance.

**Comment by ste-gr** [comment]
Hi @MarcSchiering,

at `struts-config` version 1.4 there is a a new attribute `struts-config / controller / maxStringLen`. Here you could set the new value.

**Example:**
struts-apps / examples / src / main / WEB-INF / upload / struts-config.xml

In this example the attribute `maxFileSize` and `inputForward` is set. To set also `maxStringLen` just add this attribute to the `controller`-tag.

Greetings
Stefan
